lib/frame.rb

```diff
@@ -0,0 +1,68 @@
+class Frame
+
+  def initialize
+    @roll_number = 0
+    @total_rolls = 2
+  end
+
+  def strike(roll)
+    if roll == 10
+      puts "You got a strike!"
+      return true
+    end
+  end
+
+  def spare
+    if ((@roll1 + @roll2) == 10)
+      puts "You got a spare!"
+      return true
+    end
+  end
+
```

Comment:
Nice use of helper methods - could they become question mark methods as they return true or nil (false)?